charts/u4a-component/values.yaml

```diff
@@ -67,7 +67,7 @@ oidcServer:
   cert:
     ipAddresses: *oidcIPs
     dnsNames: *dnsNames
-  image: u4a-component/oidc-server:v0.1.0-20230209
+  image: kubebb/oidc-server:v0.1.0
```

Comment:
add a suffix `-ce` for all u4a images  like https://hub.docker.com/r/kubebb/bff-server-ce